### ISSUE TITLE
Hope to support multi-directory synchronization

### ISSUE THREAD

**Issue by StrollStars** [omnivore-app/obsidian-omnivore#21]
Can it be synchronized to different directories according to tags?

**Comment by david-vaclavek** [comment]
Hello, I'd also like such feature a I use  a couple of subcategories in my Obsidian vault.

Btw: @StrollStars have you tried to use Auto Note Mover after the Omnivore import process on its folder with any success? Would such processes chaining work?

**Comment by StrollStars** [comment]
@david-vaclavek Thank you for your solution. Since I need Local Images Plus to download pictures, I don’t use Auto Note Mover. I’m afraid that too many plug-in operation notes will cause problems, but I think it’s worth a try.
Another alternative is to use tags instead of folders, but it will be mixed in with other tags. I thought of using nested tags, but I don't think it's as convenient as folders.

**Comment by david-vaclavek** [comment]
@StrollStars tried it, and it sort of works for my needs.

If they could only add the ["Allow to Overwrite an Existing Note on Prompt"](https://github.com/farux/obsidian-auto-note-mover/issues/51) feature, that would be perfect for me.

**Comment by StrollStars** [comment]
@david-vaclavek Are you using automatic mode?

**Comment by david-vaclavek** [comment]
@StrollStars Yes, movement is triggered automatically if the rule is applicable.

**Comment by StrollStars** [comment]
@david-vaclavek Is it possible to solve this problem manually?
